app/code/Magento/Directory/Model/ResourceModel/Country/Collection.php

```diff
@@ -187,6 +187,19 @@ public function loadByStore($store = null)
         return $this;
     }
 
+    /**
+     * Load allowed countries for all stores
+     *
+     * @param null|int|string|\Magento\Store\Model\Store $store
+     * @return \Magento\Directory\Model\ResourceModel\Country\Collection
+     */
+    public function loadByAllStore($store = null)
```

Comment:
1. This is [backward incompatible change](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/) as this class has `@api` annotation and you introduced a public method.
1. Does that mean that for any customer there will be shown all countries that even maybe absent for a specific store?